src/test/harness.ts

```diff
@@ -32,16 +32,34 @@ export default async function initialize(options?: TaskClientOptions) {
     );
   }
 
+  const useAadAuth = process.env.USE_AAD_AUTH;
```

Comment:
i would have figured this would be optional and you would use aad auth if it's set. does it need the required check below?